app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -81,7 +82,7 @@ def update_in_provider_queue(params)
 
   def delete_in_provider
     with_provider_object(&:destroy!)
-    self.class.send('refresh', manager)
+    self.class.send('refresh', self)
```

Comment:
I think the `delete` should still be a full refresh, right?  I don't know if deletes can be targeted since it might have to be removed from other relationships.